app/questionnaire/questionnaire_schema.py

```diff
@@ -165,8 +166,8 @@ def _create_min_max_map(
         for answer in answers:
             value = answer.get(min_max, {}).get("value")
 
-            if isinstance(value, float | int):
-                value_length = len(str(value))
+            if isinstance(value, float | int | Decimal):
+                value_length = len(str(int(value)))
```

Comment:
Comment here too about decimals would be good

Reply:
(i.e. The value is being rounded to an integer as decimals are accounted for already in jinja filters)